perun-core/src/main/java/cz/metacentrum/perun/core/entry/MembersManagerEntry.java

```diff
@@ -1233,6 +1233,32 @@ public RichMember createSponsoredMember(PerunSession session, Vo vo, String name
 		return membersManagerBl.getRichMember(session, membersManagerBl.createSponsoredMember(session, vo, namespace, name, password, sponsor, true));
 	}
 
+	@Override
+	public RichMember setSponsoredMember(PerunSession session, Vo vo, User userToBeSponsored, String namespace, String password, User sponsor)
+		throws PrivilegeException, AlreadyMemberException, LoginNotExistsException, PasswordCreationFailedException,
+		ExtendMembershipException, WrongAttributeValueException, ExtSourceNotExistsException, WrongReferenceAttributeValueException,
+		UserNotInRoleException, PasswordStrengthException, InvalidLoginException {
+		Utils.checkPerunSession(session);
+		Utils.notNull(vo, "vo");
+		Utils.notNull(userToBeSponsored, "userToBeSponsored");
+		Utils.notNull(namespace, "namespace");
+		Utils.notNull(password, "password");
+
+		log.info("setSponsoredMember(vo={},namespace='{}',displayName='{}',sponsor={}", vo.getShortName(), namespace, userToBeSponsored.getFirstName() + " " + userToBeSponsored.getLastName(), sponsor == null ? "null" : sponsor.getId());
```

Comment:
```suggestion
		log.debug("setSponsoredMember(vo={},namespace='{}',displayName='{}',sponsor={}", vo.getShortName(), namespace, userToBeSponsored.getFirstName() + " " + userToBeSponsored.getLastName(), sponsor == null ? "null" : sponsor.getId());
```
This should be probably debug, even though it was in the old method.